README.md

```diff
@@ -19,37 +19,24 @@ Usage:
 └─────────────────────────┘
 
  -h, --help    Display help for any command OR this message
- -v, --version Print ncm CLI version
 
-ncm details <module>
-ncm details <module@version>
-
-ncm report
-ncm report <directory>
-  -l, --long
-  -c --compliance
-  -s --security
-  --filter=<value>
-
-ncm signin (interactive)
- -g, --github Sign in via GitHub account
- -G, --google Sign in via Google account
-
-ncm signout
+...
```

Comment:
Are you removing the output because it's cumbersome to keep up to date or too long?

Reply:
If for updating, we need to provide examples here anyway in other places..

If for length, maybe cut it off later so the reader gets a better feel for the cli from this.

Reply:
it's long and duplicates what we are saying here

Maybe it could be removed entirely. A real TOC would be better.

Reply:
I agree, better to remove altogether than only show that snippet.